packages/config/src/Config.ts

```diff
@@ -27,6 +27,7 @@ export interface Config {
   global?: boolean
   dir: string
   bin: string
+  ignoreDepScripts?: boolean
```

Comment:
Oh, and will this also exclude workspace dependencies? Maybe we can make an exception for workspace deps, as they are a little different than npm dependencies.